doc(file-upload.md): documentation for file upload from HTML form

nverwer opened the pull request:
This file documents how to upload files from an HTML form. Examples (HTML, javascript, OpenAPI, XQuery) are given.

joewiz commented:
Thank you! Would you please enrich your fenced code blocks with the appropriate language codes for syntax highlighting? See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting.

nverwer commented:
I have added the language codes to the fenced code blocks. Somehow, git wanted to merge something and that did not go right, so I deleted and reinstated the branch in my fork. But now Github cannot find it anymore.

nverwer commented:
I cannot re-open this PR, and will open a new one. My apologies for the PR pollution.
